private/bufpkg/bufremoteplugin/bufremotepluginref/bufremotepluginref.go

```diff
@@ -145,10 +145,11 @@ func newInvalidPluginIdentityStringError(s string) error {
 }
 
 func parsePluginReference(reference string, revision int) (PluginReference, error) {
-	name, version, ok := strings.Cut(reference, ":")
-	if !ok {
+	index := strings.LastIndex(reference, ":")
+	if index == -1 {
 		return nil, fmt.Errorf("plugin references must be specified as \"<name>:<version>\" strings")
 	}
+	name, version := reference[:index], reference[index+1:]
```

Comment:
Is this always a `remote/owner/name`? If so, nit:
```suggestion
	pluginFQN, version := reference[:index], reference[index+1:]
```